.github/workflows/packages-build.yml

```diff
@@ -0,0 +1,139 @@
+          # variable to be able to use those tools.
+          if [ -f ~/dependencies_cache/paths.txt ]; then
+            cat ~/dependencies_cache/paths.txt
+            cat ~/dependencies_cache/paths.txt >> $GITHUB_PATH
+          fi
+
+          # Remove checksum file, it is not needed anymore.
+          rm checksum.txt
+
+
+
+      - name: Build.
+        shell: bash
+        run: |
+          python agent_build/build_package.py ${{ matrix.package_type }} build --output-dir agent-build-output
+          ls -al agent-build-output/${{ matrix.package_type }}/*.${{ matrix.package_type }}*
+
+
+      - name: Save the package.
+        uses: actions/upload-artifact@v2
+        with:
+          name: "package-${{ matrix.package_type }}"
+          path: |
+            agent-build-output/${{ matrix.package_type }}/*.${{ matrix.package_type }}*
+
+
+
+      - name: Save the frozen test script.
+        uses: actions/upload-artifact@v2
+        with:
+          name: "package-test-${{ matrix.package_type }}"
+          path: |
+            agent-build-output/${{ matrix.package_type }}/frozen_binary_test/package_test*
+
+
+
+
+  test:
+    needs: build
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: ["ubuntu-20.04"]
+        docker-image: ["ubuntu:14.04", "ubuntu:16.04", "ubuntu:18.04", "ubuntu:20.04"]
+        package_type: ["deb"]
+        python-version: [3.8]
+        include:
+          - os: ubuntu-20.04
+            docker-image: "centos:7"
+            package_type: "rpm"
+            python-version: "3.8"
+          - os: ubuntu-20.04
+            docker-image: "centos:8"
+            package_type: "rpm"
+            python-version: "3.8"
+          - os: ubuntu-20.04
+            docker-image: "amazonlinux:2"
+            package_type: "rpm"
+            python-version: "3.8"
+          - os: ubuntu-20.04
+            package_type: "tar"
+            docker-image: "ubuntu:14.04"
+            python-version: "3.8"
+          - os: windows-2019
+            package_type: "msi"
+            python-version: "3.8"
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - uses: actions/download-artifact@v2
+        with:
+          name: "package-${{ matrix.package_type }}"
+          path: ~/package
+
+      - uses: actions/download-artifact@v2
+        with:
+          name: "package-test-${{ matrix.package_type }}"
+          path: ~/package_test
+
+      - name: Test.
+        shell: bash
+        run: |
+          ls -al ~/package_test
+          chmod +x ~/package_test/package_test*
+          ls ~/package/*.${{ matrix.package_type }}*
+          python3 tests/package_tests/package_test_runner.py \
+            --docker-image "${{ matrix.docker-image }}" \
+            --package-path ~/package/*.${{ matrix.package_type }}* \
+            --package-test-path ~/package_test/package_test* \
+            --scalyr-api-key "0v7dW1EIPpglMcSjGywUdgY9xTNWQP/kas6qHEmiUG5w-"
```

Comment:
Let's please remove this here, add it as a repo secret (if not already set) and revoke / delete this key in the UI (I know it's a testing account, but still).

Reply:
Oh, I just wanted to ask someone to add this secret to the repo. I can't do that due to the permissions.